docs/how-to/installation/install-on-kubernetes.txt

```diff
@@ -0,0 +1,166 @@
+  the bucket. The pods launched by the Determined master must have read, write, and delete access to the bucket.
+  For example, when launching their `GKE nodes <https://cloud.google.com/sdk/gcloud/reference/container/node-pools/create>`__
+  users need to specify ``--scopes=storage-full`` to configure proper GCS access.
+
+
+Configuring Default Pod Specs (Optional)
+----------------------------------------
+
+As described in the :ref:`determined-on-kubernetes` guide, when tasks (e.g., experiments, notebooks) are started
+in a Determined cluster running on Kubernetes, the Determined master launches pods to execute these tasks.
+The Determined helm chart makes it possible to set default pod specs for all CPU and GPU tasks.
+The defaults can be defined in ``values.yaml`` under ``taskContainerDefaults.cpuPodSpec`` and
+``taskContainerDefaults.gpuPodSpec``. For examples of how to do this and a description of permissable
+fields please see the :ref:`specifying custom pod specs <custom-pod-specs>` guide.
+
+
+Configuring an External Database (Optional)
+-------------------------------------------
+
+By default, the Helm chart will deploy an instance of Postgres on the same
+Kubernetes cluster where Determined itself is deployed. If this is undesirable,
+users can configure the Helm chart to use an external Postgres database by setting
+``db.hostAddress`` to the IP address of their database. If ``db.hostAddress`` is
+configured, the Determined Helm chart will not deploy a database.
+
+
+Installing Determined
+---------------------
+
+Once finished making configuration changes in ``values.yaml`` and ``Chart.yaml``, Determined is ready to
+be installed. To install Determined run:
+
+.. code::
+
+    helm install <name for your deployment> --wait charts/determined
+
+``charts/determined`` is a relative path to where the :download:`Determined Helm Chart </helm/determined-helm-chart.tgz>`
+is located. Helm will install Determined within the default namespace. If you wish to install Determined into a
+non-default namespace, add ``-n <namespace name>`` to the command shown above.
+
+Once the installation has completed, instructions will be displayed for
+discovering the IP address assigned to the Determined master. The IP address can
+also be discovered by running ``kubectl get services``.
+
+
+Upgrading Determined
+--------------------
+
+To upgrade Determined or to change a configuration setting, first make the appropriate
+changes in ``values.yaml`` and ``Chart.yaml``, and then run:
+
+.. code::
+
+    helm upgrade <name for your deployment> --wait helm/charts/determined
+
+
+Uninstalling Determined
+-----------------------
+
+To uninstall Determined run:
+
+.. code::
+
+    # Please note that if the Postgres Database was deployed by Determined it will
+    # deleted as part of this, permanently removing all records of your experiments.
+    helm delete <name for your deployment>
+
+    # If there were any active tasks when uninstalling, this commmand will
```

Comment:
Non-blocking: I wonder if we should recommend that users do a graceful shutdown inside Determined first.

Reply:
Added a sentence here about this.